dev/com.ibm.ws.security.openidconnect.server/resources/com/ibm/ws/security/openidconnect/server/internal/resources/OidcServerMessages.nlsprops

```diff
@@ -209,4 +209,32 @@ ID_TOKEN_MISSING_REQUIRED_CLAIMS=CWWKS1647E: The ID token is not valid because i
 ID_TOKEN_MISSING_REQUIRED_CLAIMS.explanation=The OpenID Connect specification requires that the claim or claims that are listed in the error message must be present in the ID token.
 ID_TOKEN_MISSING_REQUIRED_CLAIMS.useraction=Ensure that the ID token contains all of the required claims.
 
-# NOTE: Stop at CWWKS1649
+BACKCHANNEL_LOGOUT_RESPONSE_NOT_SUCCESSFUL=CWWKS1648E: A back-channel logout request sent by the {0} OpenID Connect provider to {1} with a logout token with claims {2} did not succeed. The response status code is {3} and the response body is [{4}].
+BACKCHANNEL_LOGOUT_RESPONSE_NOT_SUCCESSFUL.explanation=The OpenID Connect client did not return a successful status code for the back-channel logout request. If the status code is 400, the logout request was not valid. If the status code is 501, logout at the OpenID Connect client failed. If the status code is 504, local logout at the OpenID Connect client succeeded but some downstream logout attempts failed.
+BACKCHANNEL_LOGOUT_RESPONSE_NOT_SUCCESSFUL.useraction=See the response status code in the message for more information. Send another logout request to retry the back-channel logout.
+
+BACKCHANNEL_LOGOUT_REQUEST_EXCEPTION=CWWKS1649E: A back-channel logout request sent by the {0} OpenID Connect provider to {1} with a logout token with claims {2} encountered an error: {3}
+BACKCHANNEL_LOGOUT_REQUEST_EXCEPTION.explanation=The back-channel logout request either timed out, threw an exception, was cancelled, or was interrupted.
+BACKCHANNEL_LOGOUT_REQUEST_EXCEPTION.useraction=See the error that is included in the message for more information. Ensure that the back-channel logout URI is responsive. Update the backchannelLogoutRequestTimeout configuration attribute in the OpenID Connect provider if the back-channel logout request needs more time.
```

Comment:
REVIEWED useraction:
Ensure that the back-channel logout URI is responsive. Update the backchannelLogoutRequestTimeout configuration attribute in the OpenID Connect provider if the back-channel logout request needs more time. For more information, see the error included in the message.